core/src/scss/utilities/placeholders/components/_card.scss

```diff
@@ -9,12 +9,13 @@
 //
 
 %card-base {
```

Comment:
As a way to deprecate and slowly remove these placeholders I was thinking we can use a mixin that calls the placeholder to provide the same result and then remove the placeholder in v6.